CONTRIBUTING.md

```diff
@@ -94,6 +94,18 @@ After the initial production release of Qiita, changes to the database schema wi
 2. We keep fully patched versions of the DBS and HTML files in the repository
 3. We keep a patch file for each patch as required in the `qiita_db/support_files/patches` directory. Note that **the patches will be applied in order based on the natural sort order of their filename** (e.g., `2.sql` will be applied before `10.sql`, and `10.sql` will be applied before `a.sql`)
 
+### Patch 91.sql
+
+In May 2024 we decided to:
+* Merge all patches into the main database, this means that there are no patches younger than 92.sql.
+* Not allow INSERT/UPDATE queries within the main *.sql patches, to minimize the possibility of unnexpected changes.
+* Added a new folder `patches/test_db_sql/` where we can store sql files that will only be applied for the test environment.
+
+Note that these changes mean:
+1. 92.sql is the current first sql file to patch the database.
+2. If you need to make changes to both the production and tests database you will need to use the *.py.
```

Comment:
sounds kludgy. What if a change was part of the test_db_sql AND independently in a .py file for production? I would think I'd want to have all the test data in one place and this would go against that.